pkg/provisioningv2/kubeconfig/manager.go

```diff
@@ -131,13 +132,19 @@ func getPrincipalID(clusterNamespace, clusterName string) string {
 	return fmt.Sprintf("system://provisioning/%s/%s", clusterNamespace, clusterName)
 }
 
-func (m *Manager) createUser(principalID, userName string) error {
+func (m *Manager) createUser(principalID, userName, clusterName string, clusterUID types.UID) error {
 	_, err := m.userCache.Get(userName)
 	if apierror.IsNotFound(err) {
 		_, err = m.users.Create(&v3.User{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:   userName,
 				Labels: labelsForUser(principalID),
+				OwnerReferences: []metav1.OwnerReference{{
+					APIVersion: "provisioning.cattle.io/v1",
```

Comment:
Nit: you can use https://github.com/rancher/rancher/blob/28345b9c6463d33b653adc78b16a3922db1e320e/pkg/apis/provisioning.cattle.io/v1/zz_generated_register.go#L35 instead of hardcode group.

Reply:
I do not believe this will work because `APIVersion` requires a string here.

Reply:
hmm, use schemeGroupVersion.String()....

Reply:
Ah there it is. Changing...